File: OrnaTrack/MyUserControls/ActiveKittyControl.vb
Public Class ActiveKittyControl
    Dim OrignalDate As String = ""
    Dim ImagePath As String = ""

    Public Property CoustName As String
        Get
            Return NameParam.Text
        End Get
        Set
            NameParam.Text = Value
        End Set
    End Property

    Public Property SrNo As String
        Get
            Return SrNoParam.Text.Split("#")(0)
        End Get
        Set
            SrNoParam.Text = "#" & Value
        End Set
    End Property

    Public Property PhNo As String
        Get
            Return PhNoParam.Text
        End Get
        Set
            PhNoParam.Text = Value
        End Set
    End Property

    Public Property CoustID As String
        Get
            Return CoustmerIdParam.Text
        End Get
        Set
            CoustmerIdParam.Text = Value
        End Set
    End Property

    Public Property ActiveKitty As String
        Get
            Return ActiveKittyParam.Text
        End Get
        Set
            'If Value = 0 Then
            '    NeverLabel.Visible = True
            '    ActiveKittyLabel.Visible = False
            '    ActiveKittyParam.Visible = False
            'Else
            ActiveKittyParam.Text = Value
            'End If
        End Set
    End Property

    Public Property SetImage As String
        Get
            Return ImagePath
        End Get
        Set
            Try
                ImageParam.BackgroundImage = Image.FromFile(Value)
            Catch ex As Exception
            End Try
            ImagePath = Value
        End Set
    End Property

    Private Sub CloseButton_Click(sender As Object, e As EventArgs) Handles CloseButton.Click
        Dispose()
    End Sub

    Public Event DetailsButton_Clicked(CustomerID As Integer)

    Public Sub DetailsButton_Click() Handles DetailsButton.Click
        RaiseEvent DetailsButton_Clicked(CoustID)
    End Sub

End Class

File: OrnaTrack/MyUserControls/ActiveKittyControl.cs
using System;
using System.Drawing;
using Microsoft.VisualBasic.CompilerServices;

namespace FirstApp
{
    public partial class ActiveKittyControl
    {
        private string OrignalDate = "";
        private string ImagePath = "";

        public string CoustName
        {
            get
            {
                return NameParam.Text;
            }
            set
            {
                NameParam.Text = value;
            }
        }

        public string SrNo
        {
            get
            {
                return SrNoParam.Text.Split('#')[0];
            }
            set
            {
                SrNoParam.Text = "#" + value;
            }
        }

        public string PhNo
        {
            get
            {
                return PhNoParam.Text;
            }
            set
            {
                PhNoParam.Text = value;
            }
        }

        public string CoustID
        {
            get
            {
                return CoustmerIdParam.Text;
            }
            set
            {
                CoustmerIdParam.Text = value;
            }
        }

        public string ActiveKitty
        {
            get
            {
                return ActiveKittyParam.Text;
            }
            set
            {
                // If Value = 0 Then
                // NeverLabel.Visible = True
                // ActiveKittyLabel.Visible = False
                // ActiveKittyParam.Visible = False
                // Else
                ActiveKittyParam.Text = value;
                // End If
            }
        }

        public string SetImage
        {
            get
            {
                return ImagePath;
            }
            set
            {
                try
                {
                    ImageParam.BackgroundImage = Image.FromFile(value);
                }
                catch (Exception ex)
                {
                }
                ImagePath = value;
            }
        }

        public ActiveKittyControl()
        {
            InitializeComponent();
            DetailsButton = _DetailsButton;
            _DetailsButton.Name = "DetailsButton";
        }

        private void CloseButton_Click(object sender, EventArgs e)
        {
            Dispose();
        }

        public event DetailsButton_ClickedEventHandler DetailsButton_Clicked;

        public delegate void DetailsButton_ClickedEventHandler(int CustomerID);

        public void DetailsButton_Click()
        {
            DetailsButton_Clicked?.Invoke(Conversions.ToInteger(CoustID));
        }

        private void DetailsButton_Click(object sender, EventArgs e) => DetailsButton_Click();

    }
}